pkgs/tools/security/gopass/default.nix

```
@@ -13,18 +13,18 @@
 
 buildGoModule rec {
```

```suggestion
buildGo118Module rec {
```
To avoid changing toplevel.